openapi-codegen/sbt-plugin/src/sbt-test/sbt-openapi-codegen/oneOf-json-roundtrip/Expected.scala.txt

```diff
@@ -96,10 +96,10 @@ object TapirGeneratedEndpoints {
   sealed trait ADTWithoutDiscriminator
   case class ValidatedObj (
     oneOf: Option[ValidatedOneOf] = None,
-    bar: ValidatedSubObj,
     foo: String,
     map: Option[Map[String, Int]] = None,
     set: Option[Set[String]] = None,
+    bar: ValidatedSubObj = ValidatedSubObj(i = "i..."),
```

Comment:
This arbitrary field re-ordering is stupid. I'm going to open another pr so that field ordering is consistent with param declaration in the yaml